gocd/gocd_test.go

```diff
@@ -18,7 +18,7 @@ var conf *config.Config
 
 func init() {
 	conf = config.NewConfig()
-	conf.SetLogLevel("info")
+	conf.SetLogLevel("debug")
```

Comment:
@frank0631 Can you actually do a quick commit to leave this as "info" for now?  We can add in the Debug option (through environment variables or whatnot) at a later point, I think.